TODO/material-design-prototype-tutorial-part-1.md

```diff
@@ -41,83 +41,83 @@ Pixate 另一项很好的方面是你可以创建自己的动作。你可以用
 *   强调色 - #D500F9
 *   登录界面背景色 - #E8F5E9
 
- Sketch 资源库的所有资源都是可以免费使用的！
+上面全部颜色都在 Sketch Assets 里，尽请使用吧！
 
-注意：在这里我假定你在接下来的构建过程中是有一定动手能力。若在下文中我略过一些内容，是因为我认为根据逻辑你们完全可以独自做到！我并没有将整个步骤完整记录下来，更深层次来说，下文应该是如何引导你去使用 Sketch 或 Pixate。但如果你觉得我确实遗漏了些重要的知识，请告知我。
+注意：我假定你在接下来的构建过程中是有一定鉴赏能力。若在下文中略过一些内容，是因为我认为按常理来说你们完全可以独自做到！本文并没有完整深入地描述，而关注于如何引导你去使用 Sketch 和 Pixate。但如果你觉得我确实遗漏了些重要的知识，请务必告知。
```

Comment:
On Tue, May 31, 2016 at 10:46 AM antonia0912 <notifications@github.com>
wrote:

> In TODO/material-design-prototype-tutorial-part-1.md
> <https://github.com/xitu/gold-miner/pull/365#discussion_r65117974>:
>
> >
> > -注意：在这里我假定你在接下来的构建过程中是有一定动手能力。若在下文中我略过一些内容，是因为我认为根据逻辑你们完全可以独自做到！我并没有将整个步骤完整记录下来，更深层次来说，下文应该是如何引导你去使用 Sketch 或 Pixate。但如果你觉得我确实遗漏了些重要的知识，请告知我。
> > +注意：我假定你在接下来的构建过程中是有一定鉴赏能力。若在下文中略过一些内容，是因为我认为按常理来说你们完全可以独自做到！本文并没有完整深入地描述，而关注于如何引导你去使用 Sketch 和 Pixate。但如果你觉得我确实遗漏了些重要的知识，请务必告知。
>
> @Sausure <https://github.com/Sausure> 我觉得这么翻译是可以的
>
> —
> You are receiving this because you were mentioned.
>
>
> Reply to this email directly, view it on GitHub
> <https://github.com/xitu/gold-miner/pull/365/files/1fa4a8e5a34871a8a5db520c2d0cf79ecf8dbb31..077a5224c7a7a072679b2b34590e77f216373268#r65117974>,
> or mute the thread
> <https://github.com/notifications/unsubscribe/AJF0BzYg7D_F0mY5K6RKc5T9IpLYh9wNks5qG6EEgaJpZM4Ij-sx>
> .
>
那可以进去下一步了咯